compose-builder/Makefile

```diff
@@ -598,9 +676,11 @@ ifeq (no-secty, $(filter no-secty,$(ARGS)))
 else
 	COMPOSE_FILES:=$(COMPOSE_FILES) -f add-security.yml
 	ifneq (mqtt-bus, $(filter mqtt-bus,$(ARGS)))
-		ifneq (zmq-bus, $(filter zmq-bus,$(ARGS)))
-		    # Using secure redis messagebus by default if MQTT/ZMQ bus not specified
-			COMPOSE_FILES:=$(COMPOSE_FILES) -f add-secure-redis-messagebus.yml
+		ifneq (nats-bus, $(filter nats-bus,$(ARGS)))
+			ifneq (zmq-bus, $(filter zmq-bus,$(ARGS)))
```

Comment:
could we just use BUS variable to do the logic or it  is hard in make file?

Reply:
I like it! Will try.

Reply:
Works and much cleaner. THX!

Reply:
Awesome!